src/test/java/bio/terra/workspace/service/iam/SamServiceTest.java

```diff
@@ -171,12 +171,18 @@ void ListPermissionsIncludesAddedUsers() {
             .build();
     RoleBinding expectedWriterBinding =
         RoleBinding.builder().role(IamRole.WRITER).users(Collections.emptyList()).build();
+    RoleBinding expectedApplicationBinding =
+        RoleBinding.builder().role(IamRole.APPLICATION).users(Collections.emptyList()).build();
+    RoleBinding expectedEditorBinding =
+        RoleBinding.builder().role(IamRole.EDITOR).users(Collections.emptyList()).build();
     assertThat(
         policyList,
         containsInAnyOrder(
             equalTo(expectedOwnerBinding),
             equalTo(expectedWriterBinding),
-            equalTo(expectedReaderBinding)));
+            equalTo(expectedReaderBinding),
```

Comment:
interesting. I thought you could make a list of the expected values and plug it in there.